website/docs/procs.md

```diff
@@ -194,5 +194,5 @@ end
 ```
 
 In general, you're better off avoiding `Proc.new` if you can. There's a
-[rubocop rule](https://www.rubocop.org/en/stable/cops_style/#styleproc) you can
+[rubocop rule](https://docs.rubocop.org/rubocop/0.92/cops_style.html#styleproc) you can
```

Comment:
The rubocop docs don't link to a "stable" version as far as I could see, so I used the latest which wasn't master.